packages/tenacity-git/tenacity-git.pacscript

```diff
@@ -0,0 +1,111 @@
+name="tenacity-git"
+url="https://github.com/tenacityteam/tenacity.git"
+build_depends="build-essential cmake git python3-pip ffmpeg"
+depends="libgtk2.0-dev libasound2-dev libavformat-dev libjack-jackd2-dev uuid-dev"
+replace="audacity"
+description="FLOSS Audacity Fork. No telemetry, crash reports and other shenanigans like that!"
+optdepends="ffmpeg: additional import/export capabilities"
+removescript="yes"
+maintainer="echometer <guacamolek@yandex.com>"
+
+pkgver() {
+    git ls-remote "${url}" master | cut -f1 
+}
+version="$(pkgver)"
+
+prepare() {
+          sudo pip3 install conan
+}
+
+build() {
+        mkdir build
```

Comment:
Missing `sudo`

Reply:
[>`sudo` not required.
-Wizard28](https://github.com/pacstall/pacstall-programs/pull/75#discussion_r675960568)

Reply:
On develop branch for 1.6, it's not required, but on master branch, 1.5., it still needs it

Reply:
At least for the CI to work

Reply:
If you plan to merge this one in the near future, add it, if not, keep it the way it is